Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.py

```diff
@@ -628,6 +647,149 @@ def get_event_incident_id_command(client: Client, args: dict) -> Tuple:
+
+    outputs = {
+        'Illusive.Event(val.eventId == obj.eventId)': {
+            'eventId': int(event_id),
+            'incidentId': int(incident),
+            'ForensicsAnalyzers': analyzers
+        }
+    }
+
+    readable_output = tableToMarkdown('Illusive Forensics Analyzers', analyzers)
+
+    return (
+        readable_output,
+        outputs,
+        incident  # raw response - the original response
+    )
+
+
+def get_forensics_triggering_process_info_command(client: Client, args: dict) -> Tuple:
+    event_id = args.get("event_id")
+    try:
+        processes = client.get_forensics_triggering_process_info(event_id)
+    except DemistoException as e:
+        if "404" in e.args[0]:
+            raise DemistoException("failed to get forensics for Event id {}".format(event_id))
+        elif "429" in e.args[0]:
+            raise DemistoException(
+                "The allowed amount of API calls per minute in Illusive Attack Management has exceeded. In case this"
+                " message repeats, please contact Illusive Networks support")
+        else:
+            raise DemistoException("{}".format(e.args[0]))
+
+    readable_output = tableToMarkdown('Illusive Triggering Processes Info', processes.get('processes'))
+    outputs = {
+        'Illusive.Event(val.eventId == obj.eventId)': {
+            'eventId': event_id,
+            'ForensicsTriggeringProcess': processes.get('processes')
+        }
+    }
+
+    return (
+        readable_output,
+        outputs,
+        readable_output  # raw response - the original response
+    )
+
+
+def get_forensics_artifacts_command(client: Client, args: dict):
+    event_id = args.get("event_id")
+    artifact_type = args.get("artifact_type", "DESKTOP_SCREENSHOT")
+    try:
+        client.get_event_incident_id(event_id)
+        artifact = client.get_forensics_artifacts(event_id, artifact_type)
+    except DemistoException as e:
+        if "404" in e.args[0]:
+            raise DemistoException("failed to get forensics for Event id {}".format(event_id))
+        elif "429" in e.args[0]:
+            raise DemistoException(
+                "The allowed amount of API calls per minute in Illusive Attack Management has exceeded. In case this"
+                " message repeats, please contact Illusive Networks support")
+        else:
+            raise DemistoException("{}".format(e.args[0]))
+
+    if len(artifact) == 0:
+        raise ValueError("file is empty")
+    return (
+        fileResult(
+            filename=f'eventId_{event_id}_DesktopScreenshots',
+            data=artifact,
+            file_type=EntryType.ENTRY_INFO_FILE
+        )
+    )
+
+
+def link_forensics_artifacts_name_command(isArtifactFile: bool, client: Client, args: dict) -> CommandResults:
+    event_id = args.get("event_id", 0)
+    if isArtifactFile:
+        outputs = {
+            'eventId': int(event_id),
+            'Artifacts': f'eventId_{event_id}_DesktopScreenshots'
```

Comment:
what's the `_DesktopScreenshots` is it the type? if so shouldn't this be configurable?

Reply:
should I add .zip ?

Reply:
Sorry i meant artifact_type as in your argument from the CLI

Reply:
It always will be zip file, not matter what is the artifact type
I've change it to:
 'Artifacts': f'eventId_{event_id}_DesktopScreenshots.zip'